src/stores/courseData.js

```diff
@@ -10,7 +10,7 @@ const DATE_FORMAT = 'YYYY-MM-DDTHH:mm:ss.SSS000Z';
 const store = new Vuex.Store({
   strict: process.env.NODE_ENV !== 'production',
   state: {
-    userYear: 0,
```

Comment:
like I said in an above comment, you could keep all of the stuff which uses `userYear` without duplicating data (the original reason for these changes) by making it a computed property in the store (here). (I think these are called `getters`)

Only if you think that's a good idea though, ofc. you've thought about this more than me at this point